web/src/app/schedules/CalendarEventWrapper.js

```diff
@@ -100,7 +99,7 @@ export default class CalendarEventWrapper extends Component {
     const { classes, event } = this.props
 
     let overrideCtrls = null
-    if (moment(event.end).isAfter(moment())) {
+    if (DateTime.fromJSDate(event.end).toUTC() > DateTime.utc()) {
```

Comment:
`toUTC` shouldn't be necessary, `#valueOf` returns the epoch timestamp which would already account for timezone differences.

Reply:
These are all identical:
- `DateTime.fromJSDate(event.end) > DateTime.local()`
- `DateTime.fromJSDate(event.end) > DateTime.utc()`
- `DateTime.fromJSDate(event.end).toUTC() > DateTime.local()`
- `DateTime.fromJSDate(event.end).toUTC() > DateTime.utc()`

But the `.toUTC()` implies it's necessary before comparing (i.e. for future code) and is just extra noise. Anyone looking at it is going to wonder why it's there, or assume they need it.

Reply:
My preference would be `DateTime.fromJSDate(event.end) > DateTime.utc()`